### ISSUE TITLE
Can't get a token.

### ISSUE THREAD

**Issue by b0nk** [pacoesteban/Rarbg-torrentapi#3]
Been trying for a few hours to get a token but I always get an error saying "Invalid API parameters!"

**Comment by eghbal** [comment]
Me too.

**Comment by pacoesteban** [comment]
Seems upstream related. I'll open an issue there.

```
http "https://torrentapi.org/pubapi_v2.php?get_token=get_token"
HTTP/1.1 200 OK
CF-RAY: 3ff0374d4d6461c3-BCN
Connection: keep-alive
Content-Encoding: gzip
Content-Type: text/html
Date: Wed, 21 Mar 2018 11:53:04 GMT
Expect-CT: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cfduid=d79c5a6d19934dbab581b0f90821234bf1521633184; expires=Thu, 21-Mar-19 11:53:04 GMT; path=/; domain=.torrentapi.org; HttpOnly
Transfer-Encoding: chunked

Invalid API parameters!
```

**Comment by pacoesteban** [comment]
Closing this as it's upstream, follow at: rarbg/torrentapi#19

**Comment by pacoesteban** [comment]
It turns out it was indeed a problem with the module not asking with the right parameters. it's fixed in version 0.1.6
